### PR TITLE
docs: change the path of members.config in quickstart.md

### PR DESCRIPTION
**Pull request by qclc**

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -55,12 +55,16 @@ To start using your KubeAdmiral, run:
...
-Please use 'kubectl config use-context kubeadmiral-member-1/kubeadmiral-member-2/kubeadmiral-member-3' to switch to the different member cluster.
+To manage your member clusters, run one of the following:
+  export KUBECONFIG=$HOME/.kube/kubeadmiral/member-1.config
+  export KUBECONFIG=$HOME/.kube/kubeadmiral/member-2.config
+  export KUBECONFIG=$HOME/.kube/kubeadmiral/member-3.config
 ```
 
-There are two kubeconfigs: `kubeadmiral.config` is the main entrypoint of a KubeAdmiral instance, while `meta.config` is only used for debugging KubeAdmiral installation with the meta cluster.
+There are three types of kubeconfig: 
+- `kubeadmiral.config` is the main entrypoint of a KubeAdmiral instance;
```
Comment (gary-lgy) [inline docs/quickstart.md:65]: Let's make this even clearer - "KubeAdmiral considers resources created in this api-server for propagation"

File: docs/quickstart.md [hunk 2]
```diff
@@ -55,12 +55,16 @@ To start using your KubeAdmiral, run:
...
+- `meta.config` is only used for debugging KubeAdmiral installation with the meta cluster;
+- `member-{1,2,3}.config` are used for managing the member clusters.
```
Comment (gary-lgy) [inline docs/quickstart.md:67]:
```suggestion
- `member-{1,2,3}.config` are used for inspecting the member clusters.
```

File: hack/make-rules/local-up.sh [hunk 3]
```diff
@@ -153,9 +153,10 @@ function print_success() {
   echo -e "  export KUBECONFIG=${HOST_CLUSTER_KUBECONFIG}"
   echo -e "\nTo observe the status of KubeAdmiral control-plane components, run:"
   echo -e "  export KUBECONFIG=${META_CLUSTER_KUBECONFIG}"
-  echo -e "\nTo manage your member clusters, run:"
-  echo -e "  export KUBECONFIG=${MEMBER_CLUSTER_KUBECONFIG}"
-  echo "Please use 'kubectl config use-context ${MEMBER_CLUSTER_NAME_PREFIX}-1/${MEMBER_CLUSTER_NAME_PREFIX}-2/${MEMBER_CLUSTER_NAME_PREFIX}-3' to switch to the different member cluster."
+  echo -e "\nTo manage your member clusters, run one of the following:"
```
Comment (gary-lgy) [inline hack/make-rules/local-up.sh:156]:
```suggestion
  echo -e "\nTo inspect your member clusters, run one of the following:"
```
`manage` suggests cluster lifecycle operations, e.g., creating / destroying / joining / removing clusters.

File: docs/quickstart.md (hunk 1 above)
Comment (qclc) [reply docs/quickstart.md]: The comment has been added, pls take a review.

File: docs/quickstart.md (hunk 2 above)
Comment (qclc) [reply docs/quickstart.md]: The content has been changed.

File: hack/make-rules/local-up.sh (hunk 3 above)
Comment (qclc) [reply hack/make-rules/local-up.sh]: The content has been changed.

**Review by gary-lgy — APPROVED** [review]

**Pull request merged**